BUILDING.md

```diff
@@ -77,12 +77,22 @@ binary artifacts are also made available to your Python environment.
 ## Running tests
 
 Testing is disabled by default and must be enabled by setting the
-`OPENASSETIO_ENABLE_TESTS` CMake variable. Assuming that the working
-directory is the repository root, we can configure the CMake build as
-follows
+`OPENASSETIO_ENABLE_TESTS` CMake variable.
+
+In addition, test fixtures can be configured such that a Python
+environment is created during the test run by setting
+`OPENASSETIO_ENABLE_PYTHON_TEST_VENV`. If this is `OFF` (default) then
+you must ensure test dependencies are installed (see below) and ensure
+OpenAssetIO is discoverable by the Python interpreter (e.g. by
+augmenting your `PYTHONPATH`) before running tests.
+
+Assuming that the working directory is the repository root, we can
+configure the CMake build as follows
 
 ```shell
-cmake -S . -B build -DOPENASSETIO_ENABLE_TESTS=ON
+cmake -S . -B build \
+  -DOPENASSETIO_ENABLE_TESTS=ON \
+  -DOPENASSETIO_ENABLE_PYTHON_TEST_VENV=ON
```

Comment:
[question] What do we think about using a preset here instead? For simplez?

Reply:
I'm less keen on that, since it adds another layer of indirection. Maybe a good idea for quick start though.